src/components/tabs/js/tabsDirective.js

```diff
@@ -131,27 +131,28 @@ function MdTabs ($$mdSvgRegistry) {
             '<md-icon md-svg-src="'+ $$mdSvgRegistry.mdTabsArrow +'"></md-icon> ' +
           '</md-next-button> ' +
           '<md-tabs-canvas ' +
-              'tabindex="{{ $mdTabsCtrl.hasFocus ? -1 : 0 }}" ' +
               'aria-activedescendant="{{$mdTabsCtrl.getFocusedTabId()}}" ' +
```

Comment:
I think it gets straight up removed. From what I can gather on the aria practices page (https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel), "When the tab list is receiving focus, places focus on the active tab element . When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is typically either the first focusable element inside the tab panel or the tab panel itself." I don't think the tablist element itself ever actually receives focus. (The example also has only the tab items receiving focus.)